### PR TITLE
Psionic Hypnosis / MindSwap Nerf

### PR DESCRIPTION
**Pull request by FoxxoTrystan**
# Description

Adds a new Psionic Power called "Psionic Hypnosis", this gives you the ability to hypnotize peoples making them do your bidding, Its takes around 30 Seconds to hypnotize someone without them moving or you moving.

Some things to note this is a psionic power, this will only affect psionics, you cannot hypnotize non-psionic things.
This power will not work on users with a mindshield.

Using dispell on a hypnotized user will remove the hypnotic effect! (The hypnotist can release their subjects freely)

You can only have 1 Subject at any time (unless changed in comp)

This power is carefully admin logged.

In short: You cannot hypnotize, mindbroken, psionic insulated, mindshielded, dead or crit peoples.

I also ask if @VMSolidus is intrested for me to port that power into EE, with ofc some changes il say and a respace.

THIS PR PATCHES MINDSAWP ADDING IT A DELAY.

---

<details><summary><h1>Media</h1></summary>
<p>

image
image
image
image

</p>
</details>

---

# Changelog

:cl:
- add: New Psionic Power: Psionic Hypnosis!
- tweak: Mindswap has now a 5 sec cooldown and cannot target mindshiedled peoples.

### CODE REVIEW

**Comment by Memeji** [comment]
Just to confirm, this is primarily an RP-focused thing?
Or how would it work mechanically?

**Comment by FoxxoTrystan** [comment]
> Just to confirm, this is primarily an RP-focused thing? Or how would it work mechanically?

Its actully been made more mechanical in mind, rp use for those who wana rp with it but its works like most psionics powers, making the target actully in your alligience, can have uses as antag or not, ect...

**Comment by FoxxoTrystan** [comment]
System uses the same way as "Mindslave" implant that is upcoming, getting hypnotzied required 30secs of long wait and has a limit, getting freed from it is easy.

RP/ERP wise uses of it follow i supose its the same way as consent OOC.

**Comment by Memeji** [comment]
I'll look into this over the weekend!

**Comment by FoxxoTrystan** [comment]
By request by Floof, this also nerfs/add changes to MindSwap, cannot mindswap mindshielded users and 5sec delay on use, so the action is not instant.
